README.md

```diff
@@ -3,13 +3,52 @@
 - List 3 reasons why asymptotic analysis may be misleading with respect to
   actual performance in practice.
 
+  
+
+  1. Input size matters becuase when doing the asymptotic analysis we focus on how the algorithm scales as the input size increases. When an algorithm takes a small input or even a medium size input the runtime can be seriously influenced by constant factors and lower order terms that have been disregarded. if I have an algorithm, $3n^3 + 2n^2 + 50n + 100$ you have a time complexity of $O(n^3)$ Then you have another algorithm with a time complexity of $O(n^2)$, $100n^2 + 50(100)+100$. If you take a small input size of $n=100$ the lower order terms in the first algorithm have a larger inpact on the operations being preformed. <br><br>
+  n=100<br> Algo $A_1$: $3(100^3)+2(100^2)+50(100)+100$ = 3,025,100 <br>
+  Algo $B_1$: $100(100^2)+50(100)+100$ = 1,005,100 <br><br>
+  n=1,000,000<br>
+  Algo $A_2$: $3(1,000,000^3)+2(1,000,000^2)+50(1,000,000)+100$ = $3.000002*10^{18}$<br>
+  Algo $B_2$: $100(1,000,000^2)+50(1,000,000)+ 100$ = $1.00000005 * 10^{14}$<br><br>
+  Lower order terms and constants:<br>
+  n=100 <br>
+  Algo $A_1$: $2(100^2)+50(100)+100$ = 25100 or .8297% of Algo $A_1$<br>
+  Algo $B_1$: $50(100)+100$ = 5100 or .5074% of Algo $B_1$ <br><br>
+  n=1,000,000<br>
+  Algo $A_2$: $2(1,000,000^2)+50(1.000,000)+100$ = $2.00005 * 10^{12}$ or .0000667% of Algo $A_2$ <br>
+  Algo $B_2$: $50(1,000,000)+100$ = $50000100$ or .00005% of Algo $B_2$<br><br>
+  You can see from the percentages that as the input size grows the lower order terms and constants have a much smaller impact on the amount of operations the algorithm does. So in practice you need to know how impactuful your lower order terms and constants will be based on individual circumstances.
+
+This also goes into seeing a algorithm with a smaller time complexity but a large constant going slower then an algorithm with a worse time complexity and smaller constant. 
```

Comment:
But this will only be true up to a certain point.

Reply:
Yes, for large datasets the asymptotic complexity is what's going to dictate which algorithm is better. But if the dataset is under that threshold like I've shown, that can be misleading between practice and theory

Reply:
Ok, so this would be the same as your reason above? Or are these two different reasons?